docs/source/contribute.rst

```diff
@@ -9,44 +9,53 @@ Researchers
-`GitHub pull requests`_ with
+We're always looking for new materials to add to this shared neuroimaging resource.  In particular, we're looking for:
 
-- samples of consent forms allowing re-distribution/deposit to
-  public archives
+- Samples of consent forms allowing re-distribution/deposit to
+  public archives 
 
-- relevant publications and discussions
+- Relevant publications and discussions
 
-- changes/recommendations for the **ultimate** consent form formulation
+- Changes/recommendations for the **ultimate** consent form formulation
+
+To add to the materials on this site, please open an issue on our `GitHub issues`_ page or send a new pull request via `GitHub pull requests`_.  Whether or not you should open an issue or make a pull request depends on the type of contribution you are making.
+
+For sample consent forms and links to relevant publications, please make submissions exclusively via the `GitHub issues`_ page.  When submitting consent forms, please include a full URL to the form and the desired filename you'd like to see it represented as on the samples_ page.  Note that for the URL you submit, `persistent URLs`_ such as `DOIs`_ are ideal, since these will not require frequent updates to this site in the event that a link moves.  If a consent form does not have a DOI associated with it, an `Internet Archive Wayback Machine`_ saved page also works well in practice.
+
+Changes to the ultimate consent form should be made via `GitHub pull requests`_.
 
 IRB committee members
 ---------------------
 
-We would welcome your feedback  very much, in particular
+We would welcome your feedback very much, in particular:
 
-- what concerns on public sharing of neuroimaging data you might have
-  if any identifiable information removed (e.g. skull stripped) and
-  research participants agreed to those terms?
+- What concerns on public sharing of neuroimaging data you might have
+  if all identifiable information is removed (e.g. skull stripped) and
+  research participants agreed to those terms.
 
-- what particular consent form composition and wording aspects would
-  you recommend? (e.g. "make it an explicit additional form requiring
+- What particular consent form composition and wording aspects would
+  you recommend (e.g. "make it an explicit additional form requiring
   a separate signature") and why?
 
 Contact information
 ===================
 
-- directly via email open-brain-consent@datalad.org
+- Directly via email at open-brain-consent@datalad.org
 - `GitHub issues`_ page
 
-.. _GitHub issues: https://github.com/datalad/open-brain-consent/issues
-.. _GitHub pull requests: https://github.com/datalad/open-brain-consent/pulls
+.. _GitHub issues: https://github.com/con/open-brain-consent/issues
+.. _GitHub pull requests: https://github.com/con/open-brain-consent/pulls
+.. _persistent URLs: https://en.wikipedia.org/wiki/Persistent_uniform_resource_locator
+.. _DOIs: https://doi.org/10.1000/182
+.. _samples:
```

Comment:
Seems to be forgotten

Reply:
I don't know why the samples link is empty.  I'll check on that.

Reply:
I'm not sure what I was trying to do there, but I've removed that and replaced it with a cross-reference earlier on in the doc.